blocks/event-community-link-component/event-community-link-component.js

```diff
@@ -0,0 +1,66 @@
+import { getLibs } from '../../scripts/utils.js';
+import { generateToolTip } from '../../utils/utils.js';
+
+const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+async function decorateField(row, type = 'text') {
+  row.classList.add('text-field-row');
+  const cols = row.querySelectorAll(':scope > div');
+  if (!cols.length) return null;
+  const [checkboxText, placeholderCol] = cols;
+  const text = placeholderCol.textContent.trim();
+
+  let input;
+
+  if (type === 'text') {
+    input = createTag('sp-textfield', {
+      id: 'community-URL-details',
```

Comment:
Nit: this stood out to me a little as I don't see many uppercase letters in id/class across Franklin projects but it's totally up to you.

Reply:
I can fix this for consistency